library/templates/subcontent/_color.html

```diff
@@ -0,0 +1,48 @@
+<!-- Swatchs -->
+{% macro swatchWithAttribute(color) %}
+  <div library-class="color-swatch--ATTRIBUTES">
+    <span library-class="color-swatch--swatch" style="background: {{ color.palette|ibmHex(color.tone) }}"></span>
+    <div library-class="color-swatch--info">
+      <p library-class="color-swatch--name">{{ color.palette }} {{ color.tone }}</p>
+      {% if color.adjectives %}
+      <ul library-class="color-swatch--attributes">
+        {% for adjective in color.adjectives %}
+        <li>{{ adjective }}</li>
+        {% endfor %}
+      </ul>
+      {% endif %}
+    </div>
+  </div>
+{% endmacro %}
+
+{% macro swatch(color) %}
+  <div library-class="color-swatch" style="background: {{ color.palette|ibmHex(color.tone) }}; color: {{color.tone|ibmTextColor }};">
+    <p library-class="color-swatch--name">{{ color.palette}} {{ color.tone }}</p>
```

Comment:
Yup, probably. If you want to comment on a specific line, you can hover over the by the line number and click the `+` to add a line note